devDocs/developerGuide.t2t

```diff
@@ -698,6 +698,23 @@ class AppModule(appModuleHandler.AppModule):
 --- end ---
 ```
 
+++ Parsing additional command line arguments in your plugin ++
+By default NVDA accepts limited set of command line arguments and shows an error for an unknown ones.
+However if you want to react to an additional arguments this is possible by adding a handler to the appropriate extension point in the addonHandler.
+Note that since command line arguments are processed just after NVDA starts your add-on needs to process them in the global plugin since app modules or other drivers may not be loaded at this stage.
```

Comment:
```suggestion
Note that since command line arguments are processed just after NVDA starts, your add-on needs to process them in the global plugin, since app modules or other drivers may not be loaded at this stage.
```